R/defaults.R

```diff
@@ -86,7 +86,7 @@ epidist.default <- function(data, formula = epidist_formula(data),
     backend = "cmdstanr", data = data, ...
   )
 
-  class(fit) <- c(class(fit), "epidist_fit")
+  class(fit) <- c(class(fit), "epidist_fit", family$name)
```

Comment:
Should this be `epidist_latent_lognormal`

Reply:
Check whether order of classes matters. Or more generally how dispatch where you have methods for multiple classes

Reply:
have you done this?